src/context.jl

```diff
@@ -247,6 +249,7 @@ end
     maxwarnings::Integer,
     debug::Bool,
     parsingdebug::Bool,
+    validate::Bool,
```

Comment:
I don't think we need to add this as a new field to `Context`, right? It should only ever be needed during the construction of the `Context` object

Reply:
that's a good point! And i see we do this for "`parsingdebug`" too (which is used in constructing the `Options`) -- i've pushed a commit removing it from the `Context` type itself.